*** phpBB3/install/install_install.php.orig	Fri Sep 17 11:28:04 2010
--- phpBB3/install/install_install.php	Fri Sep 17 11:43:31 2010
***************
*** 1956,1965 ****
  
  		$template->assign_vars(array(
  			'TITLE'		=> $lang['INSTALL_CONGRATS'],
! 			'BODY'		=> sprintf($lang['INSTALL_CONGRATS_EXPLAIN'], $config['version'], append_sid($phpbb_root_path . 'install/index.' . $phpEx, 'mode=convert&amp;language=' . $data['language']), '../docs/README.html'),
  			'L_SUBMIT'	=> $lang['INSTALL_LOGIN'],
  			'U_ACTION'	=> append_sid($phpbb_root_path . 'adm/index.' . $phpEx, 'i=send_statistics&amp;mode=send_statistics'),
  		));
  	}
  
  	/**
--- 1956,1967 ----
  
  		$template->assign_vars(array(
  			'TITLE'		=> $lang['INSTALL_CONGRATS'],
! 			'BODY'		=> sprintf($lang['INSTALL_CONGRATS_EXPLAIN'], $config['version'], append_sid($phpbb_root_path . 'install/index.' . $phpEx, 'mode=convert&amp;language=' . $data['language']), '../docs/README.html') . "<h3>This last step has been automatically performed by the Cherokee Market. The directory has been renamed and made inaccessible for your convenience!</h3>",
  			'L_SUBMIT'	=> $lang['INSTALL_LOGIN'],
  			'U_ACTION'	=> append_sid($phpbb_root_path . 'adm/index.' . $phpEx, 'i=send_statistics&amp;mode=send_statistics'),
  		));
+ 
+ 		exec ('( sleep 2 && mv ' . dirname(__FILE__) . ' ' . dirname(__FILE__).'.market_moved && chmod 000 '. dirname(__FILE__).'.market_moved > /dev/null 2>&1 ) & ');
  	}
  
  	/**
