--- drush/commands/sql/sql.drush.inc.orig	2011-03-15 12:22:16.000000000 +0100
+++ drush/commands/sql/sql.drush.inc	2011-03-15 12:26:33.000000000 +0100
@@ -180,7 +180,10 @@
 function _drush_sql_connect($db_spec = NULL) {
   switch (_drush_sql_get_scheme($db_spec)) {
     case 'mysql':
-      $command = 'mysql';
+      $command = getenv ('MYSQL_PATH');
+	 if (empty($command)) {
+	    $command = 'mysql';
+	 }
       break;
     case 'pgsql':
       $command = 'psql';
@@ -265,7 +268,7 @@
     $db_spec = _drush_sql_get_db_spec();
   }
   $database = $db_spec['database'];
-  
+
   // Get the setting of --result-file.  If the user
   // has set $options['result-file'] = TRUE, then we
   // will generate an SQL dump file in the same backup
@@ -372,7 +375,7 @@
       }
       break;
   }
-  
+
   if (drush_get_option('gzip')) {
     if ($file) {
       // Gzip the result-file
@@ -491,10 +494,10 @@
     default:
       $query = 'SHOW TABLES;';
   }
-  
+
   $filename = drush_save_data_to_temp_file($query);
   $exec = drush_sql_build_exec($db_spec, $filename);
-  
+
   // Actually run this prep query no matter if in SIMULATE.
   $old = drush_get_context('DRUSH_SIMULATE');
   drush_set_context('DRUSH_SIMULATE', FALSE);
@@ -846,7 +849,11 @@
   $scheme = _drush_sql_get_scheme($db_spec);
   switch ($scheme) {
     case 'mysql':
-      $exec = 'mysql';
+      $exec = getenv ('MYSQL_PATH');
+	 if (empty($exec)) {
+	    $exec = 'mysql';
+	 }
+
       $exec .= _drush_sql_get_credentials($db_spec);
       $exec .= ' ' . drush_get_option('extra');
       $exec .= " < $filepath";
